CMakeLists.txt

```diff
@@ -1,16 +1,17 @@
-# This file is generated by piemaker. DO NOT EDIT this file directly.
-
-cmake_minimum_required(VERSION 3.15)
+# This file is generated by piemaker 0.9.0. DO NOT EDIT this file directly.
+cmake_minimum_required(VERSION 3.21)
 set(PROJECT_NAME vectorcxx-vector)
 set(CMAKE_CXX_STANDARD 17)
 
+message(STATUS "#################### PROJECT ${PROJECT_NAME} ####################") 
+
 # https://vcpkg.readthedocs.io/en/latest/users/integration/#cmake
 if(DEFINED ENV{VCPKG_ROOT}) 
     message(STATUS "vcpkg found VCPKG_ROOT=$ENV{VCPKG_ROOT}")
     # https://github.com/microsoft/vcpkg/blob/master/docs/users/integration.md#changing-the-triplet-1
 
     if(DEFINED ENV{VCPKG_DEFAULT_TRIPLET})
-        message(STATUS "using VC_DEFAULT_TRIPLET from environment $ENV{VCPKG_DEFAULT_TRIPLET}")
+        message(STATUS "using VCPKG_DEFAULT_TRIPLET from environment $ENV{VCPKG_DEFAULT_TRIPLET}")
```

Comment:
There are multiple changes for building related stuffs: ENVs, presets, etc. Do we have any update on the README? Or these changes are transparent to developers so we don't need to do any change in development?

Reply:
It doesn't matter. I generate the Cmake file with latest piemaker and update some dependencies